blocks/app-download/app-download.css

```diff
@@ -0,0 +1,105 @@
+.block.app-download > div {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+    padding: 1.5rem 0;
+    position: relative;
+    gap: 1rem;
+}
+
+.block.app-download h2 {
+  color: var(--link-color-orange);
+  font-weight: var(--font-weight-semibold);
+  padding-bottom: 1.25rem;
+  margin: 0;
+  text-align: center;
+  font-size: 1.625rem;
+}
+
+.block.app-download h3 {
+  font-weight: var(--font-weight-medium);
+  padding-bottom: 1.25rem;
+  text-align: center;
+  font-size: 1.125rem;
+}
+
+.block.app-download > div p {
+  font: var(--font-weight-regular) 1rem / 1.5rem Poppins, sans-serif;
```

Comment:
@ravverma fallback is "Poppins Fallback"

Reply:
done